src/braket/aws/aws_quantum_task.py

```diff
@@ -133,6 +133,10 @@ def create(
         )
         if tags is not None:
             create_task_kwargs.update({"tags": tags})
+        if isinstance(task_specification, Circuit) and task_specification.is_parameterized:
+            raise ValueError(
+                f"Cannot execute parameterized circuit. " f"Invalid circuit: ({task_specification})"
```

Comment:
Maybe list out the unbound parameters

Reply:
Going to update this code for the execution PR. For now, as we are entirely blocking execution and just producing the object code, the message shall indicate that the execution of parameterized circuits is not supported.

Reply:
Ascii output will include parameters that are not set.